apps/src/dance/Dance.js

```diff
@@ -501,14 +501,18 @@ Dance.prototype.onPuzzleComplete = function (result, message) {
   // Stop everything on screen.
   this.reset();
 
-  const danceMessage = message ? danceMsg[message]() : '';
-
+  // Assign danceMessage the value of the message key if the key exists.
+  // Otherwise, assign it an empty string.
+  const danceMessage = message && danceMsg[message] ? danceMsg[message]() : '';
```

Comment:
nit: i think we can simplify this to `danceMsg[message] ? danceMsg[message]() : ''`?

Reply:
Thanks! Will update.